### PR TITLE
enabled ability to set 'extractScripts' for error pages

### PR DESCRIPTION
**Pull request by giuliaghisini**

### CODE REVIEW

**Review by nzambello — APPROVED** [review]
LGTM

**Review by tiberiuichim — APPROVED** [review]
For people that follow the Volto changelog but not all of our discussions, maybe it's good to add some info here as well: https://docs.voltocms.com/configuration/settings-reference/#server-specific-serverconfig

**Comment by giuliaghisini** [comment]
> For people that follow the Volto changelog but not all of our discussions, maybe it's good to add some info here as well: https://docs.voltocms.com/configuration/settings-reference/#server-specific-serverconfig

yes, done!

File: docs/source/configuration/settings-reference.md [hunk 1]
```diff
@@ -10,17 +10,17 @@ This list is still incomplete, contributions are welcomed!
...
+Navigation levels depth used in the navigation endpoint calls. Increasing this is useful for implementing fat navigation menus. Defaults to `1`.
 
 ### defaultBlockType
 
 !!! block ""
-    The default block type in Volto is "text", which uses the current DraftJS-based implementation for the rich text editor. Future alternative rich text editors will need to use this setting and replace it with their block type. The block definition should also include the `blockHasValue` function, which is needed to activate the Block Chooser functionality. See this function signature in [Blocks > Settings](../blocks/settings.md).
+The default block type in Volto is "text", which uses the current DraftJS-based implementation for the rich text editor. Future alternative rich text editors will need to use this setting and replace it with their block type. The block definition should also include the `blockHasValue` function, which is needed to activate the Block Chooser functionality. See this function signature in [Blocks > Settings](../blocks/settings.md).
```
Comment (sneridagh) [inline docs/source/configuration/settings-reference.md:18]: @tiberiuichim @giuliaghisini This is due to a prettier or a linter? I think it should be indented, isn't it?

Comment (tiberiuichim) [reply docs/source/configuration/settings-reference.md:18]: AFAIK it should be indented

**Review by sneridagh — APPROVED** [review]

**Pull request merged**